lib/resources/service.rb

```diff
@@ -134,9 +134,9 @@ def select_service_mgmt # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticCom
         else
           SysV.new(inspec, service_ctl || '/usr/sbin/service')
         end
-      elsif %w{redhat fedora centos}.include?(platform)
+      elsif %w{redhat fedora centos parallels virtuozzo}.include?(platform)
```

Comment:
Yes I think that's a better idea, I already submitted the fix to train: https://github.com/chef/train/pull/166

But this was explicitly set as well so fixed it in both places. I can rebase and switch to use your suggestion if needed